RioCode-Old/src/main/java/frc/robot/Commands/StateCommands/EjectCoralPose.java

```diff
@@ -0,0 +1,32 @@
+package frc.robot.Commands.StateCommands;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj2.command.Command;
+import frc.robot.Robot;
+import frc.robot.subsystems.RobotMode;
+
+public class EjectCoralPose extends Command{
+
+    private double timer = 0;
+
+    public EjectCoralPose() {
+        addRequirements(Robot.arm, Robot.climber, Robot.manipulator, Robot.elevator);
+    }
+
+    @Override
+    public void initialize() {
+        timer = Timer.getFPGATimestamp();
+        Robot.manipulator.setSpeed(0.25);
```

Comment:
Why not just schedule a parallel deadline command with a wait as the deadline? Would be cleaner than tracking a timeout ourselves. In fact, this whole command could be the parallel deadline group. Remember that the Pose commands can just be a sequential or parallel command group themselves. Maybe we should make a very simple "SetRobotMode" command we can chain for stuff like this.